platform/consul/controller_test.go

```diff
@@ -131,8 +131,7 @@ func TestInstances(t *testing.T) {
 			}
 		}
 		if !found {
-			t.Errorf("Instances() did not match by tag => %q, want tag {%q:%q}",
-				inst, filterTagKey, filterTagVal)
+			t.Errorf("Instances() did not match by tag {%q:%q}", filterTagKey, filterTagVal)
```

Comment:
Go vet decided it doesn't like code to format structs using `%q`.

Reply:
%v then?

Reply:
Yep, that works. Which seems silly, because the behavior for `%q` is basically equivalent to `"%v"`. Whatever, I just removed the struct formating entirely instead.